README.md

```diff
@@ -6,7 +6,7 @@
 [![Test Coverage][coveralls-image]][coveralls-url]
 [![Gittip][gittip-image]][gittip-url]
 
-Node.js middleware for serving a favicon.
+Node.js middleware for serving the browser default favicon.  
```

Comment:
This file is not limited to only web browsers, so this addition doesn't really make sense; as an example, using most RSS readers, they will show the `favicon.ico` file, but aren't considered to be browsers. They also show up on shortcuts to sites on at least Windows desktops.